modules/cli/src/main/scala/scala/cli/commands/SharedCompilationServerOptions.scala

```diff
@@ -58,7 +58,10 @@ final case class SharedCompilationServerOptions(
   @Group("Compilation server")
     bloopDefaultJavaOpts: Boolean = true,
   @Group("Compilation server")
-    bloopJavaOpt: List[String] = Nil
+    bloopJavaOpt: List[String] = Nil,
+  @Group("Compilation server")
+  @HelpMessage("Bloop global options file")
+    bloopGlobalOptionsFile: String = (os.home / ".bloop" / "bloop.json").toNIO.toAbsolutePath().toString(),
```

Comment:
`.toNIO.toAbsolutePath` shouldn't be needed, `os.Path`s are already absolute and normalized:
```suggestion
    bloopGlobalOptionsFile: String = (os.home / ".bloop" / "bloop.json").toString,
```